apps/web/src/pages/api/trpc/$.ts

```
@@ -51,13 +56,20 @@ const createContextRest = (
 });
 /* c8 ignore stop */
 
+type Callback = Extract<
+	ServerRouteMethodRecordValue<
+		ServerFileRoutesByPath["/api/trpc/$"]["parentRoute"],
+		"/api/trpc/$",
+		undefined
+	>,
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
+	Function
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)